src/connectors/source.rs

```diff
@@ -696,9 +682,11 @@ where
             }
             SourceMsg::Cb(cb, id) => Ok(self.handle_cb(cb, id).await),
             #[cfg(test)]
-            SourceMsg::Ping(sender) => {
-                self.ctx
-                    .swallow_err(sender.send(()).await, "Error sending Pong");
+            SourceMsg::Synchronize(sender) => {
+                self.ctx.swallow_err(
+                    sender.send(()).map_err(|_| "send err"),
+                    "Error sending Pong",
```

Comment:
Perhaps change `Error sending Pong` to something like `Error synchronizing with source` otherwise the intent/meaning is a little lost in the update as Ping/Pong is the old terminology but `Pong` persists under the renamed enum

Reply:
👍 good catch, didn't rename the error when renaming the message to something close to what it does.